src/controllers/orderStatus.controller.js

```diff
@@ -0,0 +1,34 @@
+import { Sales } from '../database/models';
+import { Orders } from '../database/models';
+
+const trackOrderStatus = async (req, res) => {
+  try {
+    const buyerId = req.user.id;
+    const selectedOrder = await Orders.findOne({
```

Comment:
All these functionalities should be created by middleware or helpers not inside controllers